billcom_integration/models/account_move.py

```diff
@@ -0,0 +1,1084 @@
+                    else:
+                        # Failed sync
+                        error_msg = f"No ID in bulk response: {result}"
+                        bill.with_context(skip_billcom_sync=True).write(
+                            {
+                                "billcom_sync_status": "sync_failed",
+                                "billcom_sync_error": error_msg,
+                            }
+                        )
+
+                        bill.message_post(
+                            body=f"<p><strong>Bill.com Bulk Update Failed</strong></p>"
+                            f"<p>No ID returned in response</p>"
+                            f"<p><em>Response: {result}</em></p>",
+                            message_type="notification",
+                            subtype_xmlid="mail.mt_note",
+                        )
+
+                        error_count += 1
+                        _logger.error(
+                            "Failed to update bill %s: %s", bill.name, error_msg
+                        )
+
+                except Exception as e:
+                    error_msg = str(e)
+                    bill.with_context(skip_billcom_sync=True).write(
+                        {
+                            "billcom_sync_status": "sync_failed",
+                            "billcom_sync_error": error_msg,
+                        }
+                    )
+
+                    bill.message_post(
+                        body=f"<p><strong>Bill.com Bulk Update Error</strong></p>"
+                        f"<p>{error_msg}</p>",
+                        message_type="notification",
+                        subtype_xmlid="mail.mt_note",
+                    )
+
+                    error_count += 1
+                    _logger.error(
+                        "Error processing bulk result for bill %s: %s",
+                        bill.name,
+                        error_msg,
+                    )
+
+            return (success_count, error_count)
+
+        except Exception as e:
+            error_detail = str(e)
+            service = self.env["billcom.service"]
+            friendly_message = service._extract_friendly_error(e)
+
+            _logger.error("Error in bulk update: %s", error_detail)
+
+            # Mark all bills as failed
+            for bill in bills:
+                bill.with_context(skip_billcom_sync=True).write(
+                    {
+                        "billcom_sync_status": "sync_failed",
+                        "billcom_sync_error": friendly_message,
+                    }
+                )
+
+            return (0, len(bills))
+
+    @api.model
+    def _sync_documents_cron(self):
+        """Cron job to sync documents from Bill.com"""
+        try:
+            config = (
+                self.env["billcom.config"]
+                .sudo()
+                .search(
+                    [("active", "=", True), ("company_id", "=", self.env.company.id)],
```

Comment:
```suggestion
                    [("company_id", "=", self.env.company.id)],
```
Search for active is redundant